.travis.yml

```diff
@@ -11,6 +11,10 @@ addons:
       - valgrind
       - check
       - python3.4
+install:
```

Comment:
Can you not overwrite the default `install` step? You should use `before_install` for installing additional dependencies.